packages/backend-defaults/src/entrypoints/rootHttpRouter/http/MiddlewareFactory.ts

```diff
@@ -137,18 +161,25 @@ export class MiddlewareFactory {
    * @returns An Express request handler
    */
   logging(): RequestHandler {
-    const logger = this.#logger.child({
-      type: 'incomingRequest',
-    });
-    const customMorganFormat =
-      '[:date[clf]] ":method :url HTTP/:http-version" :status :res[content-length] ":referrer" ":user-agent"';
-    return morgan(customMorganFormat, {
-      stream: {
-        write(message: string) {
-          logger.info(message.trimEnd());
+    const logger = this.#logger;
+    return morgan(
+      (tokens: TokenIndexer, req: IncomingMessage, res: ServerResponse) => {
+        const meta = getLogMeta(tokens, req, res);
+        const message = getLogMessage(tokens, req, res);
+        return JSON.stringify({ meta, message });
```

Comment:
Hmm, this seems unfortunate, surely there must be a way to wire this up so that we don't this?